boards/nucleo-l073/include/periph_conf.h

```diff
@@ -175,10 +175,16 @@ static const spi_conf_t spi_config[] = {
 /** @} */
 
 /**
- * @name    ADC configuration
+ * @name   ADC configuration
  * @{
  */
-#define ADC_NUMOF           (0)
+#define ADC_CONFIG {             \
```

Comment:
The backslashes could be vertically aligned

Reply:
done